scripts/busybox.sh

```diff
@@ -47,6 +47,13 @@ while true; do
 done
 
 :	${usrdir:="${PWD}"/usr}
+
```

Comment:
Yes, that will not work at all; you should use in the top directory /usr/share/mkinitramfs-ll; it's now fixed as said above in 65beac0 (for the installed files).